Documentation/Types/_Properties/_CreationOptions.rst.txt

```diff
@@ -17,7 +17,22 @@
         :default: false
         :Examples: :ref:`types-example-creation-option`
 
+        ..  versionadded:: 13.0
+
+        If true, clicking on the new element wizard directs the user back to
+        the page module directly instead of showing the edit form from the form
+        engine. Can be overridden with page TSconfig option
+        :confval:`saveAndClose <t3tsconfig:mod-wizards-newcontentelement-wizarditems-group-elements-name-saveandclose>`.
+
+    ..  confval:: defaultValues
+        :name: types-creationOptions-defaultValues
+        :Path: $GLOBALS['TCA'][$table]['types'][$type]['creationOptions']['defaultValues']
+        :type: array
+        :Examples: :ref:`types-example-creation-option`
+
+        ..  versionadded:: 13.0
+
         If true, clicking on the new element wizard directs the user back to
         the page module directly instead of showing the edit form from the form
-        engine. Can be overridden with page TSconfig
-        :confval:`newContentElement.wizardItems.[group].elements.[name] <t3tsconfig:mod-wizards-newcontentelement-wizarditems>`.
+        engine. Can be overridden with page TSconfig option
+        :confval:`tt_content_defValues <t3tsconfig:mod-wizards-newcontentelement-wizarditems-group-elements-name-tt-content-defvalues>`.
```

Comment:
In your linked changelog I found following quote: "The option has been renamed from
`tt_content_defValues` to `defaultValues`:". 

```suggestion
        :confval:`defaultValues <t3tsconfig:mod-wizards-newcontentelement-wizarditems-group-elements-name-tt-content-defvalues>`.
```

Reply:
Yes but the TSconfig option has not been renamed. See https://github.com/TYPO3-CMS/backend/blob/main/Classes/Controller/ContentElement/NewContentElementController.php#L386